static/src/stylesheets/module/email/_form.scss

```diff
@@ -7,6 +7,11 @@
     // adjustments, off grid
     width: calc(100% + 8px);
     margin-left: -4px;
+
+    @include mq($from: mobileLandscape, $until: tablet) {
+        margin-right: 50%;
+        width: calc(50% + 8px);
```

Comment:
seems like it might be almost free: https://twitter.com/asanders/status/669116090908336129